docker/docker.bzl

```diff
@@ -30,55 +30,65 @@ load(":push.bzl", "docker_push")
 # The release of the github.com/google/containerregistry to consume.
 CONTAINERREGISTRY_RELEASE = "v0.0.11"
 
-def docker_repositories():
+def docker_repositories(excludes=[], overrides={}):
```

Comment:
In python there are issues with using mutable objects as default vals in functions, does that apply to skylark?